### ISSUE TITLE
Support for HuggingFaceM4/idefics2-8b as a vision model

### ISSUE THREAD

**Issue by OLH21** [ollama/ollama#3677]
### What model would you like?

Can you add support for the various new promising Idefics V2 from huggingface
https://huggingface.co/HuggingFaceM4/idefics2-8b
https://huggingface.co/blog/idefics2
it's a multimodal with OCR and vision (chatty will follow soon)

**Comment by insinfo** [comment]
Yes! And while someone is (hopefully) at it, maybe also add the new sota:
https://huggingface.co/TIGER-Lab/Mantis-8B-siglip-llama3
